src/components/Error/ErrorDetail.tsx

```diff
@@ -0,0 +1,61 @@
+import { ApolloError, ServerError, ServerParseError } from "@apollo/client";
+import ExpandMoreIcon from "@mui/icons-material/ExpandMore";
+import {
+  Accordion,
+  AccordionDetails,
+  AccordionSummary,
+  Alert,
+  AlertTitle,
+  Box,
+  Typography,
+} from "@mui/material";
+import React from "react";
+
+interface Props {
+  error?: any;
+  additionalDetails?: string;
+}
+
+export const ErrorDetail: React.FC<Props> = ({ error, additionalDetails }) => {
+  if (!error) {
+    return null;
+  }
+
+  // helpful shortcut - if it's a 401, you'll just need to login again
+  if (
+    error instanceof ApolloError &&
+    error.networkError &&
+    "statusCode" in error.networkError &&
+    error.networkError.statusCode === 401
+  ) {
+    return (
+      <Box mt={2}>
+        <Alert severity="warning">
+          <AlertTitle>401 Forbidden</AlertTitle>A forbidden error was returned.
+          You may need to log out and back in again.
+        </Alert>
+      </Box>
+    );
+  }
+
+  return (
+    <Box mt={2}>
```

Comment:
Debated including technical details here, what do you think @tackley?

Reply:
We need to trade off over-confusing the user with what we need to debug, diagnose and support.

I think I'd probably log to the console with console.error, but not show the detail of the error in the UI.